_data/language.yaml

```diff
@@ -23,3 +23,7 @@ array:
   - name: C++
     url: //github.com/opentracing/opentracing-cpp
     icon: ''
+
+  - name: Contributions
+    url: //github.com/opentracing-contrib
```

Comment:
@pavolloffay this looks good to me after it points to https://github.com/opentracing-contrib/meta as @wu-sheng mentioned. Thanks.